docs/other/dotnet.md

```diff
@@ -5,44 +5,55 @@ TOCTitle: .NET Core
 ContentId: AFFD7BDB-925E-4D02-828D-4E14360C70DA
 PageTitle: .NET Core and Visual Studio Code
 DateApproved: 2/10/2018
-MetaDescription: Visual Studio Code has great support for writing and debugging .NET Core applications.
+MetaDescription: Get started writing and debugging .NET Core apps with Visual Studio Code.
 MetaTags:
 - .NET Core
 - ASP.NET Core
 - C#
 ---
 # Using .NET Core in Visual Studio Code
 
-[.NET Core](https://docs.microsoft.com/dotnet/articles/welcome) gives you a blazing fast and modular platform for creating server applications that run on Windows, Linux and macOS. Use Visual Studio Code with the C# extension to get a powerful editing experience with full support for C# IntelliSense (smart code completion) and debugging.
+[.NET Core](https://docs.microsoft.com/dotnet/articles/welcome) provides a fast and modular platform for creating server apps that run on Windows, Linux, and macOS. Use Visual Studio Code with the C# extension to get a powerful editing experience with [C# IntelliSense](https://docs.microsoft.com/visualstudio/ide/visual-csharp-intellisense) (smart code completion) and debugging.
 
-## Getting Started
+## Get started
 
-1. Install [.NET Core](https://microsoft.com/net/core).
-2. Install the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) from the VS Code Marketplace.
+Install the following prerequisites:
 
-## Hello World
+1. [.NET Core](https://microsoft.com/net/core).
+2. The [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) from the VS Code Marketplace.
 
-If you'd like to get started with a simple "Hello World" program on .NET Core, follow the steps below:
+## Create a "Hello World" app
 
 1. Initialize a C# project:
 
-  * Open the command prompt (or terminal).
-  * Navigate to the folder where you'd like to create the C# project.
-  * Type `dotnet new console`.
-  * This creates a `Program.cs` file in your folder with a simple "Hello World" program already written.
+  * Open the command shell and navigate to the folder you'd like to create the app.
```

Comment:
To be part of step 1, this and following lines need to be indented an additional space

Reply:
It's called "Terminal" or "Integrated Terminal" in VS Code, seems like it would be clearer to use that terminology here.

Reply:
Grammatically this sentence is incomplete:  "... to the folder you'd like to create the app in." or if you wanted to be formal "in which you'd like to create the app."